vcf2fhir/json_generator.py

```diff
@@ -97,6 +98,13 @@ def _add_region_studied(
         general_logger.debug("Region Uncallable %s", nocall_region[chrom])
         fhir_helper.add_regionstudied_obv(
             ref_seq, region_studied[chrom], nocall_region[chrom])
+    elif region_studied is not None and len(region_studied) == 0:
+        if conversion_region and not conversion_region[chrom].empty:
```

Comment:
Is there are reason for this condition to be a separate `if`, Instead of combining all the conditions in `elif`
```suggestion
    elif region_studied is not None and len(region_studied) == 0
        and conversion_region and not conversion_region[chrom].empty:
```